### DISCUSSION TITLE
2022/04/06/%E6%95%B0%E8%AE%BA%E8%AE%B2%E4%B9%89%E4%B9%A0%E9%A2%98%E6%91%98%E5%BD%953/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#17] [Announcements]
# 2022/04/06/%E6%95%B0%E8%AE%BA%E8%AE%B2%E4%B9%89%E4%B9%A0%E9%A2%98%E6%91%98%E5%BD%953/

第一题 \begin{array}{c}   证明：若n\ge 5, 2\le b\le n,则\\b-1|[\frac{(n-1)!}{b}] \end{array}证明：  注意到：如果$b\le n-1$，命题显然成立，所以只需要证明： n-1|[\frac{(n-1)!}{n}]为此，考虑以下情况：   Case 1: n是合数，设$n=pq$，我们将只需证明，在这种情况下，$\frac{

http://localhost:4000/2022/04/06/%E6%95%B0%E8%AE%BA%E8%AE%B2%E4%B9%89%E4%B9%A0%E9%A2%98%E6%91%98%E5%BD%953/

**Comment by Westlifers** [comment]
现在用latex写一个公式还是要挺久的，慢慢练

**Reply by ghost** [reply]
没事,一起写博客，加油！

**Reply by ghost** [reply]
## 坚持三年 ##
